### PR TITLE
dashboard: check ClientUrls length

### PR DESCRIPTION
**Pull request by HunDunDM**
Signed-off-by: Zheng Xiangsheng <hundundm@gmail.com>

### What problem does this PR solve? 

#2323 

### What is changed and how it works?

Check the length of ClientUrls of all members. If it is 0, it is considered that members are not ready.

### Check List 

Tests 

 - Manual test

### CODE REVIEW

**Comment by rleungx** [comment]
The panic happened inside `needResetAddress` actually

File: pkg/dashboard/manager.go [hunk 1]
```diff
@@ -115,7 +115,20 @@ func (m *Manager) updateInfo() {
 
 	var err error
 	if m.members, err = cluster.GetMembers(m.srv.GetClient()); err != nil {
-		log.Error("failed to get members")
+		log.Warn("failed to get members")
```
Comment (rleungx) [inline pkg/dashboard/manager.go:118]: How about printing the error?

Comment (HunDunDM) [reply pkg/dashboard/manager.go:118]: ok

**Pull request merged**